project/BlenderWang9487/README.rst

```diff
@@ -0,0 +1,106 @@
+========================
+DummyML
+========================
+
+
+
+Basic Information
+=================
+
+A dummy ML library for NSD course term project.
+
+Here is my Github repository:
+`DummyML <https://github.com/BlenderWang9487/DummyML.git>`_
+
+Problem to Solve
+================
+
+I want to build a machine learning library based on some mathematical theories.
+
+There are many easy-to-use hybrid frameworks that built for deep learning, 
+but fewer for other traditional ML algorithms. So I'm going to build one.
+
+Some regression, classification and clustering models (like Gaussian linear 
+regression, naive Bayes classifier, Kmeans, etc.) will be provided by this
+library. (There will be atleast three approaches for each mission. One of them
+is a "dummy" approach which can be treated as the baseline.)
+
+Prospective Users
+=================
+
+People who are interested in machine learning and want to try some different 
```

Comment:
Can you name one or two (applicational) problems that your system can solve?

Reply:
Could you point out where you add the problems?

Reply:
I add it [here](https://github.com/BlenderWang9487/nsdhw_22sp/blob/BlenderWang9487-proposal-submission/project/BlenderWang9487/README.rst#prospective-users), in the commit [4c133ca](https://github.com/yungyuc/nsdhw_22sp/pull/111/commits/4c133ca6b8427d5361aa3e5500098623e6d9b83e).